src/library-authoring/common/context/SidebarContext.tsx

```diff
@@ -168,16 +167,16 @@ export const SidebarProvider = ({
     });
   }, []);
 
-  const openUnitInfoSidebar = useCallback((usageKey: string) => {
+  const openContainerInfoSidebar = useCallback((usageKey: string) => {
     setSidebarComponentInfo({
```

Comment:
@pomegranited Reading `setSidebarComponentInfo` seemed strange to me the first time. Can you refactor this so it doesn't get confused with components?
```suggestion
    setSidebarItemInfo({
```